model/project.go

```diff
@@ -315,6 +315,16 @@ func (l *ModuleList) IsIdentical(m manifest.Manifest) bool {
 	return reflect.DeepEqual(manifestModules, projectModules)
 }
 
+func (l *ModuleList) GetModuleByName(find string) (*Module, error) {
```

Comment:
kind of a strange variable name since "find" is really a verb; maybe just "name"?